README.md

```diff
@@ -23,3 +23,13 @@ $ yarn add -D @guardian/types
 ```
 $ npm test
 ```
+
+### Publish to NPM
+
+Bump the version in `package.json`.
```

Comment:
It _looks_ like the convention in this project is to bump the version in branch, merge, and then publish to NPM from `main`? If that's correct perhaps I should be more explicit here?

Reply:
I don't think there *has* been a convention so far. This could be an opportunity to establish one. Do you have an opinion on what the process *should* be?

Perhaps @sndrs @SiAdcock @gtrufitt have some thoughts too?

Reply:
Cool, I was just going off what I saw in previous PRs but you're right this is a good place to have the conversation! I don't feel super strongly, but personally I like the release bump/tag etc to happen outside the context of a feature branch (mostly to avoid collisions if there are multiple feature branches open simultaneously). So I'd do the version bump and publish on `main`, and for the tag to exist on the same commit which did the version bump.

Reply:
would recommend using [`np`](https://www.npmjs.com/package/np) for manual publishing. it helps you bump packages and avoid a lot of pitfalls as part of the process

Reply:
With Source we essentially do this:

> do the version bump and publish on main, and for the tag to exist on the same commit which did the version bump

However thanks to @sndrs I'm going to look into doing this:

> would recommend using np for manual publishing.

😎 👍

Reply:
Oh cool `np` looks great, I hadn't come across it before :+1:

Reply:
> do the version bump and publish on main

Does this mean pushing to `main`?

Reply:
@JamieB-gu yep that's correct